py/DOCS.md

```diff
@@ -0,0 +1,73 @@
+# @configu/py
```

Comment:
what is this file doing?

Reply:
it's the "index" for the auto-generated docs.
<img width="1474" alt="image" src="https://github.com/configu/configu/assets/129258356/0c73bad0-3272-41c8-83cd-3b6ff6691e8c">